corehq/apps/locations/management/commands/migrate_new_location_fixture.py

```diff
@@ -33,10 +46,26 @@ def handle(self, *args, **options):
         # 4. Update domains that need to stay on hierarchical with enabled legacy toggle to be able to access conf
         # and update their location configuration to use hierarchical fixture for now
         for domain in domains_to_stay_on_hierarchical_fixture:
-            print "Setting HIERARCHICAL_LOCATION_FIXTURE toggle for domain: %s" % domain
-            HIERARCHICAL_LOCATION_FIXTURE.set(domain, True, NAMESPACE_DOMAIN)
-            print "Enabling legacy fixture for domain: %s" % domain
-            enable_legacy_fixture_for_domain(domain)
+            if not dry_run:
+                print "Setting HIERARCHICAL_LOCATION_FIXTURE toggle for domain: %s" % domain
+                HIERARCHICAL_LOCATION_FIXTURE.set(domain, True, NAMESPACE_DOMAIN)
+            else:
+                print "HIERARCHICAL_LOCATION_FIXTURE toggle would be set for domain: %s" % domain
+
+            if not dry_run:
+                print "Enabling legacy fixture for domain: %s" % domain
+                location_configuration = LocationFixtureConfiguration.for_domain(domain)
+                print "Current Configuration, Persisted: %s, Use Flat fixture: %s, Use Hierarchical fixture %s" % (
+                    not location_configuration._state.adding, location_configuration.sync_hierarchical_fixture,
+                    location_configuration.sync_flat_fixture)
```

Comment:
Won't the `location_configuration` always be missing for these domains?  They're 
```python
domains_to_stay_on_hierarchical_fixture = (set(domains_having_locations) -
                                           set(domain_with_flat_fixture_enabled))
```
So if the flat fixture toggle isn't enabled, they shouldn't have a fixture config, right?